src/aiperf/dataset/synthesis/anonymize.py

```
@@ -0,0 +1,210 @@
+    total_skipped = 0
+
+    with open(input_file, encoding="utf-8") as f:
+        for line_num, line in enumerate(f, 1):
+            line = line.strip()
+            if not line:
+                continue
+            try:
+                data = orjson.loads(line)
+                record = RawConversationRecord.model_validate(data)
+                records.append(record)
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)